featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -83,6 +84,14 @@ def test_calc_feature_matrix(es):
                                  instance_ids=range(17),
                                  cutoff_time=17)
 
+    error_text = "cannot use Dask DataFrame for cutoff_time: "\
```

Comment:
let's make this a separate test case so it runs independently of the other checks in this larger test case